clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureNetworkClient.groovy

```diff
@@ -213,9 +214,15 @@ class AzureNetworkClient extends AzureBaseClient {
     }
   }
 
-  void createSubnet(AzureCredentials creds, String resourceGroupName, String virtualNetworkName, String subnetName, String addressPrefix = '10.0.0.0/24') {
+  void createSubnet(AzureCredentials creds, String resourceGroupName, String virtualNetworkName, String subnetName,
+                    String addressPrefix = '10.0.0.0/24', String securityGroupName) {
```

Comment:
Can we remove the default value for the addressPrefix since it is not the last parameter anymore?